packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -223,6 +217,48 @@ export default util.createRule<Options, MessageIds>({
       };
     }
 
+    /**
+     * Creates a fixer that adds a "public" keyword with following spaces
+     */
+    function getMissingAccessibilitySuggestions(
+      node:
+        | TSESTree.MethodDefinition
+        | TSESTree.PropertyDefinition
+        | TSESTree.TSAbstractMethodDefinition
+        | TSESTree.TSAbstractPropertyDefinition
+        | TSESTree.TSParameterProperty,
+    ): TSESLint.ReportSuggestionArray<MessageIds> {
+      function fix(
+        accessibility: TSESTree.Accessibility,
+        fixer: TSESLint.RuleFixer,
+      ): TSESLint.RuleFix | null {
+        if (node.decorators && node.decorators.length > 0) {
+          const lastDecorator = node.decorators[node.decorators.length - 1];
+          const nextToken = sourceCode.getTokenAfter(lastDecorator);
+          if (!nextToken) {
+            return null;
```

Comment:
I expect it is _probably_ impossible, because I can't think of any situation where there wouldn't be any tokens after the last decorator, but it feels wrong to simply `!` the output of `getTokenAfter` just because I believe that (perhaps erroneously), so I was trying to code defensively. But that also means I can't think of any test cases to add test coverage here. What would you recommend doing?

Reply:
I say `!` it. If there are decorators they must be decorating something for this to be valid syntax.

In an ideal world, `sourceCode.getTokenAfter` would know that if it's given a node that must have an after, the returned token is defined. If we eventually get rich enough types the `!` will be linted as unnecessary.